ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -444,6 +441,18 @@ def check_mode(self, mode: str):
     # Private functions
     # ------------------------------------------ #
 
+    @lru_cache
+    def __get_cvp_inventory(self):
+        """
+        __get_cvp_inventory Method to get inventory data from CloudVision
+
+        Returns
+        -------
+        list
+            Data from CloudVision
+        """
+        return self.__cv_client.api.get_inventory(start=0, end=0)
+
     # Updated as per issue #365 to set default search with hostname field
     @lru_cache
     def __get_device(self, search_value: str, search_by: str = Api.device.HOSTNAME):
```

Comment:
Is this function still needed?

Reply:
Yes. It is being called by a few other functions.